installer/Migration/V5_0_0/Migration.php

```diff
@@ -281,6 +281,23 @@ public function up(): void
             ->executeQuery();
 
         $this->cleanUniqueIdTable();
+
+        $q = $this->createQueryBuilder()
+            ->select('filter_field.filter_field_id')
+            ->from('ohrm_filter_field', 'filter_field')
+            ->where('filter_field.name = :filterField')
+            ->setParameter('filterField', 'include');
+
+        $filterFieldId = $q->executeQuery()->fetchOne();
+        $this->createQueryBuilder()
+            ->update('ohrm_selected_filter_field', 'selected_filter_field')
+            ->set('selected_filter_field.filter_field_order', ':newFilterFieldOrder')
+            ->where('selected_filter_field.filter_field_order = :filterFieldOrder ')
+            ->andWhere('selected_filter_field.filter_field_id = :filterFieldId')
+            ->setParameter('newFilterFieldOrder', 1)
+            ->setParameter('filterFieldOrder', 0)
```

Comment:
`0` is this the only order defined. don't saved other e.g. `1`, `2`

Reply:
![FilterOrder](https://user-images.githubusercontent.com/97490773/165782625-fa138561-6ced-4a02-86aa-4604be47b4b9.png)
![FilterOrde](https://user-images.githubusercontent.com/97490773/165782633-8c49ee85-ed22-4f49-a9ed-c7f043e54d81.png)

Reply:
I created a pim report on 5x after upgrade it sets filter_field_order to 1 and they work well for edit. the zero was set only in 4x